circuits/circom/processMessages.circom

```diff
@@ -91,12 +91,6 @@ template ProcessMessages(
     // The state root before it is processed
     signal input currentStateRoot;
 
-    // topup signals
```

Comment:
The associated tests should be updated to ensure that this change hasn't affected any invariants. The suggestions for this issue have been applied but in order to ensure that the behavior we descrcibe no longer is a problem, a test case should be written